Modules/_collectionsmodule.c

```diff
@@ -991,11 +1076,19 @@ deque_reverse(dequeobject *deque, PyObject *unused)
     Py_RETURN_NONE;
 }
 
-PyDoc_STRVAR(reverse_doc,
-"D.reverse() -- reverse *IN PLACE*");
+/*[clinic input]
+_collections.deque.count
+
+    deque: dequeobject
+    v: object
+    /
+
+Return number of occurrences of v
```

Comment:
Add a period, and most methods of the class name it `value`. This also applies to the old document. I think it's b etter to should keep the consistent.
```suggestion
Return number of occurrences of value.
```